src/components/Form/ControlledCombobox.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
```

Comment:
I think this is just a question how we define what "controlled" means. Does it mean that I, as a user of this library, am controlling the state of the component via prop? Or is the component controlling a wrapped component? My thinking was the later but i'd be ok with the first too. In this case, we'd have to swap the names.
As for the `...Raw`: this only affects the `closeOnClickOutside` property (which only exists for backwards compatibility with the `Picklist`) and can always be overwritten when using the `Combobox`. So instead of creating a `ControlledComboboxRaw` one could always just do a `<ControlledCombobox closeOnClickOutside={false} ...>`.

Reply:
See https://reactjs.org/docs/forms.html#controlled-components

Reply:
Yes that would work. We already have some `Controlled` components though, it would probably better to have this consistent and either rename all of them or stick with `Controlled`

Reply:
I worked on the naming and also got rid of the `...Raw` component, because it's only purpose was to provide a default for the `ClickOutside` component.